vividus-plugin-datetime/src/main/resources/spring.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:util="http://www.springframework.org/schema/util"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd
+               http://www.springframework.org/schema/util https://www.springframework.org/schema/util/spring-util.xsd"
+       default-lazy-init="true">
+
+    <bean class="org.vividus.bdd.expression.DateExpressionProcessor">
+        <property name="locale" value="${bdd.date-expression-processor.locale}">
+        </property>
+    </bean>
+    <bean id="dateValidationSteps" class="org.vividus.bdd.steps.DateValidationSteps">
+        <property name="softAssert" ref="jsonSoftAssert" />
```

Comment:
I think `jsonSoftAssert` was used in DateValidationSteps mistakenly, I do see that it is not actually related to this PR, but it makes to fix: `jsonSoftAssert` -> `softAssert`